lib/webauthn/credential_options.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "securerandom"
+
+module WebAuthn
+  class CredentialOptions
+    CHALLENGE_LENGTH = 32
```

Comment:
should we allow challenge length configuration as long as we're on it? I mean, the standard states it should be at least 16, do you think it might be worth to allow them pass in the challenge length and validate it's bigger than 16 bytes?

Not that I would like to encourage people to use shorted and less secure challenges but some people might want to do it for some reason...?

Reply:
> some people might want to do it for some reason...?

I think I would prefer to wait for someone to open an issue with a good reason to do that before providing that as an option.